.ci/aws/Jenkinsfile

```diff
@@ -59,9 +59,9 @@ pipeline {
                     def config = ".ci/aws/aws_ofi_nccl_pr_ci.yaml"
                     def num_instances = 4
                     def p3dn_lock_label = "p3dn-1-4node"
-                    def p3dn_region = "us-east-1"
```

Comment:
why is this change in your PR?

Reply:
The CI definitions come from the repo itself, so there is no way to fix CI without rebasing atop a commit that contains the CI fix. Seth requested that I rebase on this to fix the tests, will provide more context offline.